appveyor.yml

```diff
@@ -169,8 +169,12 @@ for:
         if [[ -n $BY_CANARY ]] && [[ -n $DOCKER_USER ]] && [[ -n $DOCKER_PASS ]];
           then echo Logging in Docker Hub; echo $DOCKER_PASS | docker login --username $DOCKER_USER --password-stdin;
         fi"
-      - sh: "pytest -vv tests/integration"
-      - sh: "pytest -vv tests/regression"
+      # warm container integration tests might be interfered by other tests using docker.
+      # To grant exclusive access to docker, here we run warm container integration tests in sequential,
+      # then run everything else in parallel.
+      - sh: "pytest -vv -k 'TestWarmContainers' tests/integration"
+      - sh: "pytest -vv -n 2 --dist loadfile -k 'not TestWarmContainers' tests/integration"
```

Comment:
Why isn't this in the makefile?

Are we sure this works as we expect? I remember long ago we ran into problems with parallel tests because build verifies the containers were removed.

How are we handling throttle from services like CFN in our tests?

Reply:
> Why isn't this in the makefile?

- [ ] Make makefile and appveyor.yaml consistent

> Are we sure this works as we expect? I remember long ago we ran into problems with parallel tests because build verifies the containers were removed.

good call out, I excluded build integration tests when I tested it locally. 

> How are we handling throttle from services like CFN in our tests?

2 workers should not hit the throttle limit


**I am gonna convert this back to draft, looks like there are concerns over build integ tests. The canary test run is scheduled later today, will update later.**